components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -167,50 +171,55 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
 
     // Local variables
     int ncol, nlev, num_qtracers;
-    view_2d_const T_mid;
-    view_2d_const z_int;
-    view_2d_const z_mid;
-    view_2d_const p_mid;
-    view_2d_const pseudo_density;
-    view_2d_const omega;
-    view_1d_const phis;
-    view_1d_const surf_sens_flux;
-    view_1d_const surf_latent_flux;
-    view_1d_const surf_u_mom_flux;
-    view_1d_const surf_v_mom_flux;
-    view_2d_const qv;
-    view_2d       qv_copy;
-    view_2d       qc;
-    view_2d       qc_copy;
-    view_2d       shoc_s;
-    view_2d       tke;
-    view_2d       tke_copy;
-    view_2d       rrho;
-    view_2d       rrho_i;
-    view_2d       thv;
-    view_2d       dz;
-    view_2d       zt_grid;
-    view_2d       zi_grid;
-    view_1d       wpthlp_sfc;
-    view_1d       wprtp_sfc;
-    view_1d       upwp_sfc;
-    view_1d       vpwp_sfc;
-    view_2d       wtracer_sfc;
-    view_2d       wm_zt;
-    view_2d       exner;
-    view_2d       thlm;
-    view_2d       qw;
-    view_2d       cloud_frac;
+    view_1d_const_double area;
```

Comment:
Side comment: I'm baffled that github shows this huge diff, when in reality is a small diff. Took me a while to realize you simply added three lines...

Reply:
I think the issue is that to line up all the view declarations, I needed each view to have an additional space..

Reply:
Ah, I see. Ok, github didn't go nuts then. Thanks!